index.html

```diff
@@ -48,13 +63,149 @@ <h3>LET'S CONNECT</h3>
             </div>
         </section>
 
+        <section class="desktop-container">
+           <div class="mobile-version">
+            <div class="mobile-image">
+              <img src="desktop4.svg" alt="" class="card-img-img">
+            </div>
+            <div class="mobile-section-div">
+              <h1>Tonic</h1>
+              <div class="canopy-div">
+                <p>Canopy</p>
+                <img src="Counter.svg" alt="" />
+                <p>Backend Dev</p>
+                <img src="Counter.svg" alt="" />
+                <p>2015</p>
+              </div>
```

Comment:
- [ ]Hi, nice job on the row with info about the work. I noticed that the width of the div containing the list is too wide making the spacing between the company name, ) the position (Full Stack Dev) and the year the project was build, separated by grey circles too much compared to the Figma design. You might want to reduce the width property of the div to make it more aligned with the design. This will improve the overall layout and appearance of your page.


| Your Design | Figma Design |
| ------------- | ------------- |
| <img width="448" alt="Screenshot 2023-05-13 000039" src="https://github.com/Developer-Kingz/my-portforlio-project/assets/52934128/788c505f-29a4-47ce-869c-662b56536f91"> | <img width="275" alt="image" src="https://github.com/Developer-Kingz/my-portforlio-project/assets/52934128/7e442964-3a1a-4634-9bb0-6db6aef8a5da">  |